.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Add plugin
+about: Request to add your plugin to the Marketplace
+title: "Add $REPOSITORY_NAME to Marketplace"
+labels: Plugin/New
+assignees: hanzei
+---
+
+<!--
+Thank you very for submitting your plugin for consideration! A review process is required to ensure your plugin adheres to the quality standard of the Marketplace. This process may take a couple of weeks depending on Mattermost staff availability and any changes that are required.
+Read https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/ before submitting your plugin.
+-->
+
+#### Summary
+<!--
+A brief description what your plugin does. Consider including screenshots to help illustrate.
+-->
+
+#### Review Commit
+<!--
+Please link to an open source repository and release that should be used for review. As Mattermost code reviews and builds all plugins itself when listing in the Marketplace, the link cannot point at an already built plugin.
+-->
+
+## Checklist
+<!--
+Go through this checklist and confirm every item. If your plugin doesn't fulfill every item, leave a comment explaining why.
+-->
+
+**Product Requirements**
+
+- [ ] The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+- [ ] The source code is available in a public git repository.
+- [ ] There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
+- [ ] The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
+- [ ] For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+- [ ] The plugin has to be out of Beta and be released with at least v1.0.0.
+- [ ] All configuration is accessible via the Mattermost UI.
```

Comment:
```suggestion
- [ ] All configuration is accessible via the Mattermost System Console.
```

Reply:
@justinegeffen The configuration doesn't have to be in the System Console. It could e.g. also happen via a Slash Command.